builtin/merged_dataset.cc

```diff
@@ -79,9 +79,12 @@ struct MergedDataset::Itl
                   [] (std::shared_ptr<Dataset> p1,
                       std::shared_ptr<Dataset> p2)
                   {
-                      return false;
```

Comment:
Please add a virtual method `getRowCount()` to `Dataset` with a default implementation that forwards to `getMatrixView()->getRowCount()`, and sort on that highest to lowest.  We don't care about the column counts at all.  We don't want to use `getRowNames()` because it's a very expensive operation.